### PR TITLE
Codex bootstrap for #2730

### PR DESCRIPTION
**Pull request by stranske**
### Source Issue #2730: “Cosmetic tests fixer” as a controlled autofix mode

Source: https://github.com/stranske/Trend_Model_Project/issues/2730

> Topic GUID: 121cf21a-71e1-5e2e-971a-678ad0f28ba2
> 
> ## Why
> - Summary
>     - Add an optional autofix mode that only does safe test‑cosmetics: import sorting, ruff quick‑fixes, and whitespace normalization in tests/. Gate remains the enforcer; this is a convenience to lower PR churn.
> 
>   - Scope
> 
>     - Extend reusable-18-autofix.yml or add a sub‑mode triggered by label autofix:clean to target only tests/**.
> 
>     - Post a short PR comment listing files changed.
> 
> ## Tasks
> - [ ] Label‑gated mode implemented
> 
> - [ ] Only tests/** is touched
> 
> - [ ] Comment summary posted
> 
> ## Acceptance criteria
> - A demo PR with the label shows cosmetic test changes only, and Gate stays green.
> 
> ## Implementation notes
> _Not provided._
> 
> ---
> Synced by [workflow run](https://github.com/stranske/Trend_Model_Project/actions/runs/18575667639).

—
PR created automatically to engage Codex.

### CODE REVIEW

**Comment by stranske** [comment]
@codex start

Codex, please derive and propose:
- Scope / key constraints
- Acceptance criteria / definition of done
- Initial task checklist to iterate through

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

**Comment by stranske** [comment]
@codex fix comments

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

**Comment by stranske** [comment]
@codex [chore: add unformatted demo to trigger autofix](https://github.com/stranske/Trend_Model_Project/pull/2771#top) just fixed issues with the autofix process and had some conflicts that needed to be resolved. I left both sets of changes in there when resolving conflicts. I want you to make sure that the goals of the recent merge on the branch I pointed to above are honored when working through fixing the mess I've made. WIth those guidelines, please fix comments.

**Comment by stranske** [comment]
@codex On the demo/autofix-applied-v2 branch I just fixed issues with the autofix process and had some conflicts that needed to be resolved here. I left both sets of changes in there when resolving conflicts. I want you to make sure that the goals of the recent merge on the branch I pointed to above are honored when working through fixing the mess I've made. WIth those guidelines, please fix comments.

**Comment by stranske** [comment]
@codex On the demo/autofix-applied-v2 branch I just fixed issues with the autofix process and had some conflicts that needed to be resolved here. I left both sets of changes in there when resolving conflicts. I want you to make sure that the goals of the recent merge on the branch I pointed to above are honored when working through fixing the mess I've made. WIth those guidelines, please fix comments.

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

**Comment by stranske** [comment]
@codex fix comments

**Comment by stranske** [comment]
@codex fix comments

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

**Comment by stranske** [comment]
@codex fix comments

**Comment by stranske** [comment]
@codex use the task list and acceptance criteria linked above to work on Issue 2730 until all acceptance criteria are met

File: .github/workflows/reusable-18-autofix.yml [hunk 1]
```diff
@@ -574,15 +824,42 @@ jobs:
...
+        env:
+          MODE: ${{ steps.fix_results.outputs.mode }}
+          FILE_LIST: ${{ steps.fix_results.outputs.file_list }}
+          CHANGED: ${{ steps.fix_results.outputs.changed }}
+          SAME_REPO: ${{ steps.same_repo.outputs.same }}
+          REMAINING: ${{ steps.fix_results.outputs.remaining_issues }}
+          NEW_ISSUES: ${{ steps.fix_results.outputs.new_issues }}
+          PATCH_PR_NUMBER: ${{ inputs.pr_number }}
+          GUARD_SKIP: ${{ steps.guard.outputs.skip }}
         run: |
```
[the remark this answers is not in the document]
Comment (stranske) [reply .github/workflows/reusable-18-autofix.yml]: @copilot, what are they talking about? There aren't quotes, double or single?

**Comment by stranske** [comment]
@codex fix comments

**Comment by stranske** [comment]
@codex fix comments
